databases/week2/homework_4.sql

```diff
@@ -0,0 +1,225 @@
+set names utf8mb4;
+
+create database homework;
+use homework;
+CREATE TABLE regions (
+	region_id INT (11) AUTO_INCREMENT PRIMARY KEY,
+	region_name VARCHAR (25) DEFAULT NULL
+);
+
+CREATE TABLE countries (
+	country_id CHAR (2) PRIMARY KEY,
```

Comment:
It's a general convention to use the name `id` for primary keys, unless it's a composite primary key, which is not the case here.